docs/compass/03-formation-templates.md

```diff
@@ -0,0 +1,51 @@
+# Formation Templates
+
+A Formation Template is the basis for the formations. 
+It establishes a contract for each formation of the given type which states what types of applications and what type of runtime is allowed to be assigned to the given formation.
+Based on their type we can process the formations differently depending on the Formation Template restrictions and specifics.
+
+
+Each Formation Template provides a list of application types that are allowed to be assigned to a formation that is based on the template. Currently, every Formation Template allows exactly one runtime type to be included in a formation.
+The template also provides additional metadata fields such as `runtimeTypeDisplayName` and `runtimeArtifactKind`. 
+The runtime type display name is a short name describing the runtime type and the runtime artifact kind is an enum with the following allowed values: `SUBSCRIPTION`, `SERVICE_INSTANCE`  and `ENVIRONMENT_INSTANCE`.
```

Comment:
Maybe we can have one empty line to separate those two paragraphs.

What do you think about something like this for runtimeTypeDisplayName

```
The `runtimeTypeDisplayName` is the brand name representing this runtime. Unlike `runtimeType`, this value is suitable for UI visualizations and external documents where a brand name that customers are familiar with should be used.
```